main.go

```diff
@@ -462,7 +487,10 @@ func (tch *tileCachingHandler) getAndCacheTile(ctx context.Context, tile tile) (
 // getAndCacheTileUncollapsed is the core of getAndCacheTile (and is used by it)
 // without the request collapsing. Use getAndCacheTile instead of this method.
 func (tch *tileCachingHandler) getAndCacheTileUncollapsed(ctx context.Context, tile tile) (*entries, tileSource, error) {
+	begin := time.Now()
```

Comment:
This begin variable is used in 3 locations in this method, but each latency being measured is distinct. Instead of declaring begin and then re-declaring it twice, you could do `Observe(time.Since(time.Now).Seconds())`, though that comes with the visual risk of parentheses forest.